lib/styles/text_styles.dart

```diff
@@ -92,7 +92,7 @@ class TextStyles {
   static TextStyle lightGreyTextSmallBold = const TextStyle(
       fontSize: ThemeFontSizes.small,
       fontWeight: FontWeight.bold,
-      color: LightThemeColors.textLightGrey);
+      color: Color(0xFF808B9B));
```

Comment:
I see this is being used in other classes. Please verify does not have an undesired result.

Reply:
Good catch. Fixed